### PR TITLE
Req #54: comment above the init_buffer, clear_buffer, get_buffer and write_to_output_buffer declarations

### PR DESCRIPTION
**Pull request by jdembdr**
In src/hints, please proceed to comment above the init_buffer, clear_buffer, get_buffer and write_to_output_buffer declarations

### CODE REVIEW

File: src/hints/mod.rs [hunk 1]
```diff
@@ -12,21 +12,56 @@ mod skip;
...
+	/// with a RwLock used by `fn test_single_entrypoint` to store the output of the `cairo_run`
+	/// command for each test entrypoint.
+	///
+	/// Before test execution, an Uuid is associated for the test, test is executed
+	/// and `cairo_run` output is captured in the `HINT_OUTPUT_BUFFER`
+	/// # Examples
+	///
+	/// Basic usage:
+	/// ```
+	///	let mut output = String::new();
```
Comment (Eikix) [inline src/hints/mod.rs:25]: this code gets executed in tests

your documentation is executed inside rust tests
you can add a `no_run` flag

/// ```no_run
/// loop {
///     println!("Hello, world");
/// }
/// `

https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html

Comment (jdembdr) [reply src/hints/mod.rs:24]: I've used ignore instead.

File: src/hints/mod.rs [hunk 2]
```diff
@@ -12,21 +12,55 @@ mod skip;
 pub use skip::*;
 
 lazy_static! {
+	/// the `HINT_OUTPUT_BUFFER` is a hasmap<Uuid,String> protected from concurrency
```
Comment (Eikix) [inline src/hints/mod.rs:15]: Hashmap

**Review by Eikix — APPROVED** [review]
after typo correction and maybe running `cargo clippy`, good to go!

File: src/hints/mod.rs (hunk 2 above)
Comment (jdembdr) [reply src/hints/mod.rs:15]: Ok, tks. modification pushed with some automatic cargo clippy rewrites.

**Pull request merged**